kolibri/plugins/learn/assets/src/views/explore-page/index.vue

```diff
@@ -13,11 +13,18 @@
       {{ topic.description }}
     </p>
 
+    <ui-select
+      :label="$tr('display')"
+      :options="filterOptions"
+      v-model="selectedFilter"
+      class="filter"
+    />
+
     <span class="visuallyhidden" v-if="subtopics.length">{{ $tr('navigate') }}</span>
 
-    <card-grid v-if="contents.length">
+    <card-grid v-if="filteredContents.length">
       <content-grid-item
-        v-for="content in contents"
+        v-for="content in filteredContents"
```

Comment:
see also https://vuejs.org/v2/guide/list.html#key - don't know if it helps here, but from the docs:

> It is recommended to provide a `key` with `v-for` whenever possible, unless the iterated DOM content is simple, or you are intentionally relying on the default behavior for performance gains.